ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -157,10 +157,14 @@ images:
   dockerfile_path: Dockerfile
   from: base-c9s-python-3.9
   to: amd-c9s-python-3.9
-- context_dir: jupyter/minimal/ubi9-python-3.9
+- context_dir: jupyter/amd/tensorflow/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: amd-jupyter-datascience-ubi9-python-3.9
+  to: amd-jupyter-tensorflow-c9s-python-3.9
+- context_dir: jupyter/amd/pytorch/ubi9-python-3.9
   dockerfile_path: Dockerfile
-  from: amd-c9s-python-3.9
-  to: amd-jupyter-minimal-c9s-python-3.9
+  from: amd-jupyter-datascience-ubi9-python-3.9
```

Comment:
This `amd-jupyter-datascience-ubi9-python-3.9` is not build (amd specific), I think you need here `jupyter-datascience-ubi9-python-3.9`

And again, once you apply the changes please run from your pc `make jobs` this command will fix some of the tests and also will autogenerate the post-submit jobs